storage/innobase/row/row0ins.cc

```diff
@@ -1663,6 +1775,20 @@ row_ins_check_foreign_constraint(
 			case DB_SUCCESS:
 				continue;
 			default:
+#ifdef WITH_WSREP
+				/* Warning message for applier FK failure */
+				if (
+#ifndef UNIV_DEBUG
+				    err != DB_LOCK_WAIT && // Lock wait is not real error
+#endif
+				    trx->is_wsrep()) {
+					wsrep_applier_log_fk(trx, foreign,
+						      "Setting ordinary shared record lock "
+						      "in the child table "
+						      "failed err: ",
+						      ut_strerr(err));
```

Comment:
The output would seem to include `err: err:`. Please cover each message in the test cases.